tests/devapps/WAM/NetCoreWinFormsWam/Form1.cs

```diff
@@ -120,14 +120,14 @@ private IPublicClientApplication CreatePca(AuthMethod? authMethod)
                     builder = ToggleOldBroker(builder, true);
                     break;
                 case AuthMethod.WAMRuntime:
-                    builder = builder.WithBrokerPreview().WithExperimentalFeatures();
+                    builder = BrokerExtension.WithWindowsBroker(builder);
```

Comment:
Why are the extension classes explicitly called here and not through the `builder`?

Reply:
The intention is to specifically test BrokerExtension.